api/src/main/java/org/apache/cloudstack/query/QueryService.java

```diff
@@ -94,8 +94,8 @@
     ConfigKey<Boolean> AllowUserViewDestroyedVM = new ConfigKey<>("Advanced", Boolean.class, "allow.user.view.destroyed.vm", "false",
             "Determines whether users can view their destroyed or expunging vm ", true, ConfigKey.Scope.Account);
 
-    static final ConfigKey<String> UserVMBlacklistedDetails = new ConfigKey<String>("Advanced", String.class,
-            "user.vm.blacklisted.details", "rootdisksize, cpuOvercommitRatio, memoryOvercommitRatio, Message.ReservedCapacityFreed.Flag",
+    static final ConfigKey<String> UserVMDenyListedDetails = new ConfigKey<String>("Advanced", String.class,
+            "user.vm.DenyListed.details", "rootdisksize, cpuOvercommitRatio, memoryOvercommitRatio, Message.ReservedCapacityFreed.Flag",
```

Comment:
not sure if this is an issue but naming convention fro settings is all lower case.

Reply:
I've fixed incorrect case change - sloppy C&P